packages/test/test-service-load/src/runner.ts

```diff
@@ -386,6 +391,20 @@ async function setupOpsMetrics(container: IContainer, logger: ITelemetryLogger,
         }
     });
 
+    let submittedSignals = 0;
+    testRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
```

Comment:
we should combine the event handlers here. not need to have two different ones on the same event name

Reply:
Done!